.github/workflows/tox.yml

```diff
@@ -126,7 +126,7 @@ jobs:
       - run: python -m pip install PyYAML click packaging
```

Comment:
Do we need these installs if we are pipx running?

Reply:
indeed we don't. Thank you for catching this !